csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/CXX_Api_Sample.cpp

```diff
@@ -14,7 +14,7 @@ int main(int argc, char* argv[]) {
 
   // initialize session options if needed
   Ort::SessionOptions session_options;
-  session_options.SetThreadPoolSize(1);
+  session_options.SetIntraOpThreadPoolSize(1);
 
```

Comment:
This is probably a good example of how the current API could be confusing. If I see this code I would assume the 1 means single threaded not 'use current thread + threadpool of 1.